src/main/java/InputProcess.kt

```diff
@@ -0,0 +1,278 @@
+import BechTools.BASE64_FORMAT
+import BechTools.BINARY_FORMAT
+import BechTools.DEC_FORMAT
+import BechTools.HEX_FORMAT
+import BechTools.NO_FORMAT
+
+// Written by Mikita 7.4.2022
+/*
+When function returned 'false' – it means FAIL
+'true' – successful result
+ */
+
+object InputProcess {
+
+    private var isToEncode: Boolean = false
+    private var isToDecode: Boolean = false
+    private var isPrintIntoFile: Boolean = false
+
+    private var inputString: String = ""
+    private var outputFilePath: String = ""
+
+    private var inputFormat: Int = NO_FORMAT // Default: DEC for encode // BASE64 for decode
+    private var outputFormat: Int = NO_FORMAT //Default: BASE64 for encode // DEC for decode
+
+    private var inputArray: MutableList<String> = ArrayList()
+    private var outputArray: MutableList<String> = ArrayList()
+
+    fun inputArgumentsExecutor(args: ArrayList<String>): Boolean {
+        if(args.size == 1) return false
+        if (!inputArgumentsInspector(args)) return false
+
+        if (isToEncode) {
+            val bechData = BechTools.convertStringToBechData(inputString, inputFormat)
+            val encrypted = Encode.encode(bechData.humanReadablePart, bechData.data)
+            if (isPrintIntoFile)
+                FileInspector.writeStringIntoFile(outputFilePath, encrypted)
+            else
+                println(encrypted)
+        } else if (isToDecode) {
+            val decoded = Decode.decodingString(inputString)
+            if (isPrintIntoFile)
+                FileInspector.writeStringIntoFile(
+                    outputFilePath,
+                    BechTools.convertBechDataToString(decoded, outputFormat)
+                )
+            else
+                println(BechTools.convertBechDataToString(decoded, outputFormat))
+        }
+        return true
+    }
+
+    private fun inputArgumentsInspector(args: ArrayList<String>): Boolean { //Mikita
+        if (args.isEmpty()) {
+            UserCommunicator.printPrompt("You didn't provide any arguments :(")
+            return false
+        }
+
+        if (isArgIsHelp(args)) return false // if user ask for a help -> stop processing
+
+        divideArray(args) // divide array into input part and output part
+
+        if (!inputArrayInspector(inputArray)) return false
+        if (!outputArrayInspector(outputArray)) return false
+
+        return true
+    }
+
+    private fun divideArray(args: ArrayList<String>) {
+        inputArray = ArrayList()
+        outputArray = ArrayList()
+        var switch = false
+        args.forEachIndexed { _, value ->
+            if (value == "-o")
```

Comment:
String constants such as "-o" and later "-e", "-d", .. could have been defined as private vals, for example:
`private val OUTPUT_FLAG = "-o"`